### PR TITLE
Docs: Add pyproject.toml snippet for custom connectors (#977)

### PR DESCRIPTION
**Pull request by simonhammes**

### CODE REVIEW

File: docs/api/connectors.md [hunk 1]
```diff
@@ -86,3 +86,14 @@ class LocalConnector(BaseConnector):
             bool: indicating success or failure.
         """
 ```
+
+## pyproject.toml
+
+In order for pyinfra to gain knowledge about your connector, you need to add the following snippet to your connector's `pyproject.toml`:
+
+```toml
+[project.entry-points.'pyinfra.connectors']
```
Comment (Fizzadar) [inline docs/api/connectors.md:95]: Amazing, long been on the list to figure out what the correct config was here!

**Review by Fizzadar — APPROVED** [review]
This is great, anything left? I think this looks good to go..

**Comment by simonhammes** [comment]
> This is great, anything left? I think this looks good to go..

Thank you! Because I left it as a draft PR? I'm not too sure why I did, the documented approach works :smile:

File: docs/api/connectors.md (hunk 1 above)
Comment (simonhammes) [reply docs/api/connectors.md:95]: Well, your hint in https://github.com/pyinfra-dev/pyinfra/issues/977#issuecomment-1551447809 was quite good :)

**Review by DrSensor — APPROVED** [review]

**Review by Fizzadar — APPROVED** [review]
Thank you @simonhammes!

**Pull request merged**